### ISSUE TITLE
Electron not opening (no errors, just wont open at all)

### ISSUE THREAD

**Issue by ColdNightHour** [electron-userland/electron-prebuilt#156]
If there is an existing Electron issue for this, please link it here:

https://github.com/atom/electron/issues

Describe your problem:
Electron won't open at all. I ran the command to install electron-prebuilt. Nothing happens when I run `electron` or `electron myapp` afterward. I am using the code found on the website under Quick Start and it won't open so it's not the code as far as I am concerned. There have been other issues regarding electron not opening but those have been with errors popping up when trying to run electron. Mine is simple: electron doesn't do anything. I don't really know why this is happening but I need it to work asap and after having spent four hours of trying to get it to work I am running out of options.

**Comment by ewnd9** [comment]
Got this too on `electron-prebuilt@1.2.4` (latest)

```js
const electron = require('electron');

electron.app.on('ready', () => {
  const win = new electron.BrowserWindow({
    show: true
  });

  win.loadURL('file://' + __dirname + '/index.html');
});
```

```
$ node_modules/.bin/electron .
$ echo $?
```

Nothing happens, electron exists with 0 code

**Comment by ewnd9** [comment]
@ColdNightHour try the previous version, it works for me

```sh
$ npm install electron-prebuilt@1.2.3
```

**Comment by ColdNightHour** [comment]
Just did that. 1.2.3 Works perfectly. Thanks.

**Comment by wwwtyro** [comment]
1.2.3 also working for me, also fails on 1.2.4. Also on Ubuntu 16.04.

**Comment by ewnd9** [comment]
`1.2.5` is ok (ref https://github.com/electron/electron/issues/6178)

**Comment by kevinsawicki** [comment]
1.2.4 definitely had issues on Linux but they should have been addressed in 1.2.5 so I believe this issue can be closed now.
